content/applications/marketing/surveys/dashboard.rst

```diff
@@ -0,0 +1,178 @@
+
+On the dashboard, all the surveys that have been created in the database are displayed in a default
+Kanban view.
+
+From left-to-right, after the survey name, the user who is responsible for it, and the month it was
+created, each line on the *Surveys* dashboard shows the following:
+
+- Number of :guilabel:`Questions` in that particular survey
+- :guilabel:`Average Duration` of the survey (how long it typically takes a participant to complete)
+- Number of :guilabel:`Registered` participants for the survey
+- Number of times that particular survey has been :guilabel:`Completed`
+- Percentage and data bar showcasing how many people have :guilabel:`Passed` or become
+  :guilabel:`Certified`
+
+.. note::
+   A half-trophy background image behind the survey name indicates that the survey is a
+   *Certification*.
+
+.. note::
+   The :guilabel:`Passed` percentage and bar **only** appears when a *Required Score* has been
+   configured for that particular survey.
+
+   The :guilabel:`Certfied` percentage and bar **only** appears if that particular survey has the
+   *Is a Certification* option enabled on the survey form.
+
+   If neither :guilabel:`Passed` nor :guilabel:`Certified` appear on the line, that indicates the
+   survey is without a *Required Score* and was not enabled with the *Is a Certification* option.
+
+- Number of :guilabel:`Courses` related to that survey, which **only** appears if more than one
+  course has been created and attached to a single survey
+
+To the far-right of those data points on the survey lines, located on the *Surveys* application
+dashboard, are a collection of buttons.
+
+Those buttons are as follows:
+
+- :guilabel:`Share`: click to reveal a :guilabel:`Share a Survey` pop-up form that can be used to
+  invite potential participants to the survey - complete with a :guilabel:`Survey Link` that can be
+  copied and sent to potential participants, and a :guilabel:`Send by Email` toggle switch.
+
+   .. image:: dashboard/share-survey-popup.png
+      :align: center
+      :alt: The 'Share a Survey' pop-up window that appears in the Odoo Surveys application.
+
+  When the :guilabel:`Send by Email` toggle is 'on' (green switch), additional fields appear, in
+  which :guilabel:`Recipients` and a :guilabel:`Subject` can be added to the email. Below that, a
+  dynamic email template, complete with a :guilabel:`Start Certification` button appears, which can
+  be modified, as well, if needed.
+
+  Once modifications are complete, click :guilabel:`Send` to send that email invite to all the email
+  addresses/contacts listed in the :guilabel:`Recipients` field.
+
+.. image:: dashboard/share-survey-popup-email-toggle.png
+   :align: center
+   :alt: The 'Share a Survey' pop-up window in Odoo Surveys with the Send by Email toggled on.
+
+.. important::
+   Clicking the :guilabel:`Share` button results in an error if the survey line has zero questions.
```

Comment:
It may be helpful to tip where users can edit the default **Mail Template**, as well as note the conditional display of fields depending on config :)
```suggestion
  .. tip::
     The default :guilabel:`Mail Template` for survey invites can be edited by navigating to
     :menuselection:`Settings --> Technical --> Email Templates` and searching for `Survey: Invite`.

  .. note::
     The :guilabel:`Send by Email` toggle switch is **not** present when the survey line has zero
     questions.

     The :guilabel:`Survey Link` only appears when the survey's *Access Mode* is set to *Anyone with
     the link*.

     The :guilabel:`Additional Emails` field only appears when the survey's *Require Login* field is
     **not** active.
```